### ISSUE TITLE
 using 0.8.5 version code with pre-trained models(COCO-R50C4-MaskRCNN-Standard.npz) to evaluate on COCO2014 val dataset, but get very poor performance like initialized model

### ISSUE THREAD

**Issue by Dataphz** [tensorpack/tensorpack#867]
Hello, 

Because my tf version is 1.4.0, so I choose the code of 0.8.5 version. And I download COCO-R50C4-MaskRCNN-Standard.npz as pre-trained model to evaluate on coco val2014 dataset. 
However, the AP performance is almost 0, just like the performance using ImageNet-R50.npz

Is there any fault? Could you guide me to slove that?
Thanks you very much in advance!

**Comment by ppwwyyxx** [comment]
Please post issues following the issue template

**Comment by Dataphz** [comment]
Thank you for your apply, here is the issue:

1. What you did:
[train command, in order to transfer/finetune] 
python train.py --gpu 0,1,2,3 --datadir /coco --load /pretrained_models/COCO-R50C4-MaskRCNN-Standard.npz 

2. What you observed, including but not limited to the __entire__ logs.
  The AP performance on COCO val2014 dataset is almost 0. And the training precision/recall almost improves, it seems that the pretrained model on COCO is same as the ImageNet-R50.npz.

4. Your environment:
  + Python python3.6
  + TF version: 1.4.0.
  + Tensorpack version: 0.8.8 (But using 0.8.5 version of FastRCNN code)

**Comment by ppwwyyxx** [comment]
Evaluating (instead of finetuning) the model will give you the correct performance.

If you are asking why finetuning performance is not what you expected, that's a machine learning question so we don't answer it. A smaller learning rate is definitely needed.

**Comment by Dataphz** [comment]
My TF version is limited to 1.4.0, so that choosing 0.8.5 version fasterrcnn code...  And i also evaluated the model, the ap is also almost 0. And i verified the model is loaded.
Do u  have the pretrained model of 0.8.5 version model?  if convenient, can u share it with me.
Thanks for your apply!

**Comment by ppwwyyxx** [comment]
No I don't have the old models.

**Comment by ppwwyyxx** [comment]
Actually with TF 1.4 the code should be able to run most of the times. It may crash for some inputs due to TF bugs.
